Makefile

```diff
@@ -247,9 +247,9 @@ ifneq ($(HAS_WEBAPP),)
 endif
 
 .PHONY: test-e2e
-test-e2e:
+test-e2e: dist
 	@echo Running e2e tests
-	MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
+	PLUGIN_BUNDLE=$(shell pwd)/dist/$(BUNDLE_NAME) MM_SERVER_PATH=${MM_SERVER_PATH} $(GO) test -v $(GO_TEST_FLAGS) -tags=e2e $(GO_PACKAGES)
```

Comment:
Do we need to use `GO_PACKAGES` or can we fall back to `./...`?

Reply:
not sure, @catalintomai? But for this PR - I did not change that logic.

Reply:
0/5 - GO_PACKAGES  is already used in the MM codebase (and amounts to the same)